### PR TITLE
Audit log docs

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: docs/audit.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Filtering
+Filtering is available using the filter bar. the filter bar works with a simple query language.
+The table fields could be filter by the following operators
+- Time - `<`,`>`,`<=`,`>=`,`=`,`!=`
+- User  - `=`,`!=`, compare by string or regex
+- Region - `=`,`!=`, compare by string or regex
+- Status - `<`,`>`,`<=`,`>=`,`=`,`!=`
+- Action - `=`,`!=`, compare by string or regex
+- Resource - `=`,`!=`, compare by string or regex
```
[the remark this answers is not in the document]
Comment (guy-har) [reply docs/audit.md:36]: @nopcoder, so what would you suggest? 
I think it's important to state two things here 
1. it's `=` and not `==` or something else 
2. regex is supported

[the remark this answers is not in the document]
Comment (nopcoder) [reply docs/audit.md:42]: 1. Agree
2. How do I specify if it is a string or regex - if there is no way - it means that it is always regex

**Review by nopcoder — CHANGES_REQUESTED** [review]

**Review by nopcoder — APPROVED** [review]
Just specify the syntax all the rest looks good.

**Review by rmoff — COMMENTED** [review]
I've made some changes, including adding a bit of preamble to the top of the doc. 

The one changes I would suggest would be under `Filtering` to add an example or two.

**Pull request merged**